go.mod

```diff
@@ -4,29 +4,26 @@ go 1.13
 
 require (
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
-	github.com/go-logr/logr v0.3.0 // indirect
```

Comment:
nit: since we are here, we could you update the `go` directive above to `go 1.16` (and re-run `go mod vendor`)

Reply:
Good catch, done